packages/orchestrator-ui-components/src/components/WFOProcessesList/processListObjectMappers.ts

```diff
@@ -0,0 +1,79 @@
+import {
+    GraphqlFilter,
+    GraphQLSort,
+    Process,
+    ProcessesResult,
+} from '../../types';
+import { ProcessListItem } from './WFOProcessList';
+
+export const mapGraphQlProcessListResultToProcessListItems = (
+    processesResult: ProcessesResult,
+): ProcessListItem[] =>
+    processesResult.processes.page.map((process) => {
+        const {
+            workflowName,
+            lastStep,
+            lastStatus,
+            workflowTarget,
+            createdBy,
+            assignee,
+            processId,
+            startedAt,
+            lastModifiedAt,
+            subscriptions,
+            product,
+            customer,
+        } = process;
+
+        return {
+            workflowName,
+            lastStep,
+            lastStatus,
+            workflowTarget,
+            createdBy,
+            assignee,
+            processId,
+            startedAt,
+            lastModifiedAt,
+            subscriptions,
+            productName: product?.name,
+            productTag: product?.tag,
+            customer: customer.fullname,
+            customerAbbreviation: customer.shortcode,
+        };
+    });
+
+// Some fields are not a key of Process, however backend still supports them
```

Comment:
This explanation raises some questions to me. Isn't the case that some fields in the process list are not a key of a Process but a key of a nested type of a key of Process? Or is that what you are saying?

Reply:
Not exactly. It is for sure "key of Process" but additonally some concatenation of the object name and field.
`product: { name: string }` -- becomes productName. Though backend did not applied any types there. Backend assumes the field is a string instead of "keyof Process".